contracts/contracts/levels/Recovery.sol

```diff
@@ -14,26 +12,24 @@ contract Recovery {
 
 contract SimpleToken {
 
-  using SafeMath for uint256;
-  // public variables
   string public name;
   mapping (address => uint) public balances;
 
   // constructor
-  constructor(string memory _name, address _creator, uint256 _initialSupply) public {
+  constructor(string memory _name, address _creator, uint256 _initialSupply) {
     name = _name;
     balances[_creator] = _initialSupply;
   }
 
   // collect ether in return for tokens
   receive() external payable {
-    balances[msg.sender] = msg.value.mul(10);
+    balances[msg.sender] = msg.value * 10;
   }
 
   // allow transfers of tokens
   function transfer(address _to, uint _amount) public { 
     require(balances[msg.sender] >= _amount);
-    balances[msg.sender] = balances[msg.sender].sub(_amount);
+    balances[msg.sender] = balances[msg.sender] * _amount;
```

Comment:
Why was subtraction changed to multiplication?

Reply:
This is an error, fixing it.